week4:JSweek2/3.js

```diff
@@ -0,0 +1,4 @@
+var x
+console.log("the value of my variable x will be: five");
+var x = (5);
```

Comment:
you don't need the parentheses, `var x = 5;` also works

Reply:
sure. I deleted the first line of the code and it still works. I do not necessarily need to declare x like that right?
in other words, is    var x = 5    enough?